Pod/Classes/UIView+AutoLayout.h

```diff
@@ -40,9 +40,21 @@
 
 - (NSArray*)pinToHorizontalEdges;
 - (NSArray*)pinToVerticalEdges;
+- (NSArray*)pinToSuperview;
```

Comment:
Like pin to all of its edges? maybe it could be `pinToSuperviewEdges` or something. `pinToSuperview` just tells me its pinned somewhere to the superview